### PR TITLE
HBASE-24762 Purge protobuf java 2.5.0 dependency

### PR DESCRIPTION
**Pull request by Apache9**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 29s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 30s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  6s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  hadoopcheck  |  14m 43s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   1m  0s |  The patch does not generate ASF License warnings.  |
|  |   |  27m  7s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.12 Server=19.03.12 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2128 |
| Optional Tests | dupname asflicense hadoopcheck xml |
| uname | Linux 816d7ca9ebc0 4.15.0-101-generic #102-Ubuntu SMP Mon May 11 10:07:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 8191fbdd7d |
| Max. process+thread count | 63 (vs. ulimit of 12500) |
| modules | C: . hbase-client hbase-common hbase-examples hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-examples/pom.xml [hunk 1]
```diff
@@ -205,6 +205,7 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
+      <version>${internal.protobuf.version}</version>
```
Comment (virajjasani) [inline hbase-examples/pom.xml:208]: We are keeping only in hbase-examples, sounds good.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  6s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 29s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 40s |  master passed  |
| +1 :green_heart: |  compile  |   3m  2s |  master passed  |
| +1 :green_heart: |  shadedjars  |   6m 25s |  branch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 15s |  root in master failed.  |
| -0 :warning: |  javadoc  |   0m 25s |  hbase-client in master failed.  |
| -0 :warning: |  javadoc  |   0m 15s |  hbase-common in master failed.  |
| -0 :warning: |  javadoc  |   0m 16s |  hbase-examples in master failed.  |
| -0 :warning: |  javadoc  |   0m 39s |  hbase-server in master failed.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 24s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m  2s |  the patch passed  |
| +1 :green_heart: |  javac  |   3m  2s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 21s |  patch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 16s |  hbase-common in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 25s |  hbase-client in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 40s |  hbase-server in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 20s |  hbase-examples in the patch failed.  |
| -0 :warning: |  javadoc  |   0m 14s |  root in the patch failed.  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 243m  3s |  root in the patch passed.  |
|  |   | 279m 16s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.12 Server=19.03.12 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2128 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 9bb4eee29aee 4.15.0-91-generic #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 8191fbdd7d |
| Default Java | 2020-01-14 |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-root.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-client.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-common.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-examples.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-hbase-server.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-common.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-client.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-server.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-hbase-examples.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/testReport/ |
| Max. process+thread count | 4332 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-client hbase-server hbase-examples . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 31s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 41s |  master passed  |
| +1 :green_heart: |  compile  |   2m 20s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 35s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 35s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 21s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 21s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 16s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 16s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 38s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 20s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  javadoc  |   0m 22s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  javadoc  |   0m 36s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  javadoc  |   0m 20s |  hbase-examples generated 0 new + 0 unchanged - 72 fixed = 0 total (was 72)  |
| +1 :green_heart: |  javadoc  |   1m 57s |  root generated 0 new + 43 unchanged - 72 fixed = 43 total (was 115)  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 332m 44s |  root in the patch failed.  |
|  |   | 367m  8s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.12 Server=19.03.12 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2128 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 66ebe61335cb 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 8191fbdd7d |
| Default Java | 1.8.0_232 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/artifact/yetus-jdk8-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/testReport/ |
| Max. process+thread count | 5554 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-client hbase-server hbase-examples . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2128/1/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**